package.json

```diff
@@ -45,18 +48,19 @@
     "express-validator": "^6.2.0",
     "helmet": "^3.21.2",
     "i18n": "^0.8.4",
+    "knex": "^0.20.1",
```

Comment:
Similar to Calvin's earlier comment, looks like maybe some things slipped in that are unrelated to this PR. Probably want to remove this in case the Sequalize PR gets merged first.

Reply:
Knex got merged in through an earlier PR